src/indexable.cr

```diff
@@ -809,6 +809,37 @@ module Indexable(T)
     index(offset) { |e| yield e } || raise Enumerable::NotFoundError.new
   end
 
+  # Returns the first element in the indexable for which the passed block
+  # is truthy, starting from the given *offset*.
+  #
+  # Accepts an optional parameter *if_none*, to set what gets returned if
+  # no element is found (defaults to `nil`).
+  #
+  # ```
+  # [1, 2, 3, 4].find { |i| i > 2 }        # => 3
+  # [1, 2, 3, 4].find(-1, 2) { |i| i < 2 } # => -1
+  # [1, 2, 3, 4].find(-1) { |i| i > 8 }    # => -1
```

Comment:
As a last suggestion I'd perhaps switch these two examples to showcase the arguments in sequence of their definition:

```suggestion
  # [1, 2, 3, 4].find(-1) { |i| i > 8 }    # => -1
  # [1, 2, 3, 4].find(-1, 2) { |i| i < 2 } # => -1
```

Reply:
done!